snugget_load.py

```diff
@@ -107,14 +108,25 @@ def addTextSnugget(appName, row, sectionID, subsectionID, filterColumn, filterID
     (str(sectionID), str(subsectionID), str(groupID), str(filterID))
   )
   snuggetID = getSnuggetID(appName, sectionID, subsectionID, filterColumn, filterID, cur);
-  # TODO: modify this to dynamically read in localized text columns and put them in localized content columns
+
+  # dynamically build list of multilingual content and language codes
+  i18n_columns = ", ".join([c for c in (_get_i18n_db_column(column_name) for column_name in row.keys()) if c is not None])
+  i18n_placeholders = ", ".join(["%s" for c in (_get_i18n_db_column(column_name) for column_name in row.keys()) if c is not None])
+  values_to_insert = (snuggetID, row["text-en"])
+  for col in i18n_columns.split(", "):
+  	values_to_insert = values_to_insert + (row[col.replace("content_", "text-")],)
```

Comment:
This looks like it is functionally the same as _get_i18n_db_column. It'd be better to pick one approach or the other and use it everywhere.